bin/clang-format

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -e
+
+die() {
+	>&2 echo "$0: ${1-Unknown error}"
+	false
+}
+
+root=$(dirname $(dirname $(readlink -f $0)))
```

Comment:
I changed from `realpath $0` to `readlink -f $0`, but otherwise this is a direct copy off of the RGeo version.